py/kubeflow/kubeflow/ci/shared_ui_tests.py

```diff
@@ -27,7 +27,7 @@ def _create_ui_tests_task(self, task_template):
         ui_tests = argo_build_util.deep_copy(task_template)
 
         ui_tests["name"] = "shared-ui-tests"
-        ui_tests["container"]["image"] = "node:12.20.1-stretch-slim"
+        ui_tests["container"]["image"] = "browserless/chrome"
```

Comment:
Should we tag to a fixed version?

Reply:
That's a very good point! Doing it now

Reply:
I pinned it to the [1.44-chrome-stable](https://hub.docker.com/layers/browserless/chrome/1.44-chrome-stable/images/sha256-06c88d84d658321f1db07ca7f3500774494adbfb9295c2131ecbccad7ba826f7?context=explore) tag. It worries me that in the base image they just install the latest nodejs. But since we'll be using a specific commit I think we'll be OK, but lets just have it in mind.

I also tried to run the tests with this tag and they run as expected